model/pack/QtiItemPacker.php

```diff
@@ -73,9 +73,8 @@ public function packItem(core_kernel_classes_Resource $item, $lang, Directory $d
         $content = $this->getXmlByItem($item, $lang);
         //load content
         $qtiParser = new QtiParser($content);
-        //validate it
-        $qtiParser->validate();
-        if (!$qtiParser->isValid()) {
+
+        if ($this->skipValidation === false && !$qtiParser->validate()) {
```

Comment:
I recommend to add this attribute to the class

Reply:
Which one?
`$this->skipValidation` - exists in parent class